app/controllers/application_controller.rb

```diff
@@ -15,8 +15,14 @@ class ApplicationController < ActionController::Base
     rescue_with_handler(e.cause) or raise(e)
   end
 
+  around_action :within_last_api_version
+
   private
 
+  def within_last_api_version
+    DIDWW::Client.with_api_version('2022-05-10') { yield }
```

Comment:
Not sure about this implementation, how about bump the API version in `config/initializers/didww.rb`?

Reply:
@Skumring what do you want to see in this file?